client/lib/plans/constants.js

```diff
@@ -1002,3 +1002,16 @@ export function getMonthlyPlanByYearly( plan ) {
 			return '';
 	}
 }
+
+export function getYearlyPlanByMonthly( plan ) {
```

Comment:
That's true. It's an improvement that is totally worth some time, but probably for another PR after we land #14761?